src/test/crimson/perf_crimson_msgr.cc

```diff
@@ -290,15 +304,15 @@ static seastar::future<> run(unsigned rounds,
   };
 
   return seastar::when_all_succeed(
-      ceph::net::create_sharded<test_state::Server>(core),
-      ceph::net::create_sharded<test_state::Client>(jobs, rounds, bs, depth))
+      ceph::net::create_sharded<test_state::Server>(core, sbs),
+      ceph::net::create_sharded<test_state::Client>(jobs, rounds, cbs, depth))
     .then([=](test_state::Server *server,
               test_state::Client *client) {
       entity_addr_t target_addr;
       target_addr.parse(addr.c_str(), nullptr);
       if (mode == perf_mode_t::both) {
-          logger().info("\nperf settings:\n  mode=server+client\n  server addr={}\n  server core={}\n  rounds={}\n  client jobs={}\n  bs={}\n  depth={}\n",
-                        addr, core, rounds, jobs, bs, depth);
+          logger().info("\nperf settings:\n  mode=server+client\n  server addr={}\n  server core={}\n  rounds={}\n  client jobs={}\n  client bs={}\n  server bs={}\n  depth={}\n",
```

Comment:
nit, could split this into multiple lines, like
```c++
logger().info("\n"
              "perf settings:\n"
              "  mode=server+client\n"
              "  server addr={}\n"
              "  server core={}\n"
              "  rounds={}\n"
              "  client jobs={}\n"
              "  client bs={}\n"
              "  server bs={}\n"
              "  depth={}\n",
```

Reply:
The options are now wrapped with struct, see https://github.com/cyx1231st/ceph/commit/b0db7d7ff912c3d5bd6d48b16fdd1144364d16d7.